formulaire/src/main/resources/public/ts/directives/question/question-type/question-type-singleanswer-radio/question-type-singleanswer-radio.directive.ts

```diff
@@ -0,0 +1,70 @@
+import {Directive, ng} from "entcore";
+import {FormElement, FormElements, Question} from "@common/models";
+import {I18nUtils} from "@common/utils";
+import {Direction} from "@common/core/enums";
+import {RootsConst} from "../../../../core/constants/roots.const";
+
+interface IQuestionTypeSingleanswerRadioProps {
+    question: Question;
+    hasFormResponses: boolean;
+    formElements: FormElements;
+    I18n: I18nUtils;
+    Direction: typeof Direction;
+}
+
+interface IViewModel extends ng.IController, IQuestionTypeSingleanswerRadioProps {
+    deleteChoice(index: number): Promise<void>;
+    isFormElementAfter(formElement: FormElement): boolean;
```

Comment:
cf commentaire si on suit les suggestions : 

```suggestion
    i18n: I18nUtils;
    direction: typeof Direction;
    deleteChoice(index: number): Promise<void>;
    isFormElementAfter(formElement: FormElement): boolean; 
```

Et encore, ça dépend si i18n et direction sont censé être "publique" 
si on l'utilise directement dans le DOM MAIS si on passe par des méthodes pour les intéractions, on peut passer en private donc pas besoin de le déclarer dans l'interface